src/commands/documents.ts

```diff
@@ -13,9 +13,6 @@ import cliProgress from 'cli-progress';
 import { faker } from '@faker-js/faker';
 import { getAlertIndex } from '../utils';
 
-const config = getConfig();
```

Comment:
as the config file might not be there, we cannot use getConfig at the top level like this anymore